### DISCUSSION TITLE
`Dynamic`, `For`/`Index` and performance

### DISCUSSION THREAD

**Discussion by stanlrt** [solidjs/solid#2230] [Q&A] [answered]
What is more performant between: 

- Having a `Dynamic` inside a `For`/`Index` loop (assume it is massive)
- Having the same `For`/`Index` loop inside each component variant rendered by the `Dynamic`

**Comment by atk** [comment, marked as answer]
`<Dynamic as={(props) => <For.../>} />` should be more performant than `<For>{() => <Dynamic />}</For>`. The performance of Dynamic is slightly worse than that of a normal component.
